tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/LockFreeExponentiallyDecayingReservoir.java

```diff
@@ -66,6 +70,7 @@
 
     private final class State {
         private final long startTick;
+        // Count is updated after samples are successfully added to the map.
         private final AtomicLong count;
```

Comment:
@carterkozak Are you looking to pull in the changes from https://github.com/carterkozak/metrics/pull/3 here as well or separately?

Reply:
It doesn’t make a significant difference so I haven’t pulled it in here. If it merges to the main PR branch, I’ll pull it into tritium as well.

Reply:
https://github.com/carterkozak/metrics/pull/3 has been approved by an upstream contributor, would you like to review it before I PR it into tritium? I'll stage it here separately since this PR is already reviewed.

Reply:
sounds good